LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -93,6 +98,28 @@ function openContent(url) {
         	cache: false
         });
 
+        // load group data
+        dialog.indicateBusy();
+        $.ajax({
+            type: 'GET',
+            url: '../api/?a=group/members',
+            success: function(res) {
+                if(res.length > 0) {
+                    for(var i in res) {
```

Comment:
The `for..in` loop often causes problems with older versions of IE that we have to support at least in the short term. It's better to use `for(var i=0; i<whatever.length; i++)` loops, which is annoying but it works.

Reply:
Addressed in commit c131